tests/code_size/hello_world_wasm.js

```diff
@@ -21,6 +21,7 @@ WebAssembly.instantiate(d.wasm, {
     k = a.d;
     h = a.b;
     var b = h.buffer;
+    new DataView(b);
```

Comment:
This should be unneeded now with the ifdefing - perhaps it is left over from before? If not, what causes it?

Reply:
Removed